stdlib/2and3/zlib.pyi

```diff
@@ -47,7 +50,7 @@ if sys.version_info >= (3,):
 else:
     def compressobj(level: int = ..., method: int = ..., wbits: int = ...,
                     memlevel: int = ..., strategy: int = ...) -> _Compress: ...
-def crc32(__data: bytes, __value: int = ...) -> int: ...
+def crc32(__data: Union[array[_T], bytes], __value: int = ...) -> int: ...
```

Comment:
The typevar doesn't seem useful here. You could write `Union[array[int], array[float], array[Text]]` instead, but that's not very useful because those are the only kinds of arrays allowed, so just array[Any]` is fine here.